docs/Examples/Badge.examples.purs

```diff
@@ -0,0 +1,40 @@
+module Lumi.Components.Examples.Badge where
```

Comment:
This file is named `examples` instead of `example`

Reply:
Oh, thanks! What's weird is that the header links were working just fine for me 😶

Reply:
Ohh, maybe they don't rely on that anymore